compiler/rustc_driver/src/pretty.rs

```diff
@@ -503,7 +502,8 @@ fn print_with_analysis(
     }
     .unwrap();
 
-    write_output(out, ofile);
+    let out = std::str::from_utf8(&out).unwrap();
```

Comment:
Can we just unwrap? Does this never fails? If that's they case why not just `from_utf8_unchecked`?

Reply:
It should never fail, but a future change may make it fail, in which case it is better to panic than to have silent UB.